src/clojure/nrepl/cmdline.clj

```diff
@@ -26,7 +28,7 @@
   "Requests that the process exit with the given `status`.  Does not
   return."
   [status]
-  (throw (ex-info nil {::kind ::exit ::status status})))
+  (throw (ex-info nil {:nrepl/kind ::exit ::status status})))
```

Comment:
I think you should add some comment here explaining why `:kind` is different.

Reply:
I'm not sure it should be, but I think I was imagining that we might not want a `kind` per-namespace.  If the clojure world had a solid expectation that `:kind` was always a type marker in `ex-data`, then I might just use that (and we could decide to anyway), but `:nrepl/kind` avoids the question while still providing a general (not ns-specific) marker.

Put another way, while the value might well deserve to be fully namespaced, having a shared/common `kind` allows unified dispatch handling in `ExceptionInfo` catch clauses, perhaps via `case`, etc.

Reply:
Oh, hah, maybe you just meant that I should include the commit message comment (similar in spirit to https://github.com/nrepl/nrepl/pull/204#discussion_r559236620) in the code...

Reply:
Perhaps something like this, unless there's a better place for it:
```
(defn exit
  "Requests that the process exit with the given `status`.  Does not
  return."
  [status]
  ;; :nrepl/kind is our shared (ns independent) ExceptionInfo discriminator
  (throw (ex-info nil {:nrepl/kind ::exit ::status status})))
```

Reply:
Yeah, that's what I had in mind.